src/index.ts

```diff
@@ -99,10 +96,5 @@ export const supplyFetchers: Record<string, SupplyFetcher> = {
     ninjazFetcher,
   "38ad9dc3aec6a2f38e220142b9aa6ade63ebe71f65e7cc2b7d8a8535434c4159":
     clayFetcher,
-  ee0633e757fdd1423220f43688c74678abde1cead7ce265ba8a24fcd43424c50: cblpFetcher,
-  "682fe60c9918842b3323c43b5144bc3d52a23bd2fb81345560d73f634e45574d":
-    newmFetcher,
-  bb4cfbe0f6be60b80f90f815e8353b93431de4df785d75350b9d214a48455242: herbFetcher,
-  "279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b":
-    snekFetcher,
```

Comment:
Why you delete this block code?

Reply:
what do @similiken need to fix to make it work?